test/no-injection-into-hooked/Makefile

```diff
@@ -23,7 +25,9 @@ golden:
 clean:
 	rm -fr $(KOMPILED)
 
-.PHONY: test-k test golden clean
+clean-execution:
```

Comment:
What does this target do?

Reply:
After running `make test-k`, running `make test-k-simplifierx` printed "nothing to be done". So I added `clean-execution:` in order to remove the files created during execution, and not the kompiled directory.